mautic/mautic
# Channel web push

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | N
| New feature?  | Y
| BC breaks?    | N
| Deprecations? | N
| Fixed issues  |  N/A

## Description

New Web Push Channel

## Steps to Test

In order to test this, your mautic install must be on an HTTPS host, otherwise you cannot use web notifications (web push requires that the pushing website be https for security, and since your mautic install is the pusher, you have to have it be https). 

For those wanting to test, we've set up an environment at dev.mautic.com. You can get the login credentials from the mautic slack chat #engineering channel. 

If you simply want to test that you're able to receive notifications, I've already setup a notification campaign that will send you a push notification when you fill out the contact form found at http://dbhurley.com/contact AFTER you've subscribed. To test, you can navigate to that page, then click the "(or web notifications)" text in the sidebar. It will open a popup window (be sure not to block it) that will ask you to subscribe to web push notifications. Say allow, then close the window. Then you may fill out the contact form on that page, and you should soon receive a web push notification from your browser.
